NEWS

```diff
@@ -67,10 +67,16 @@ Breaking Changes
   If you see opportunities to extend ``repo-info.json`` with further information,
   please get in touch.
 
+- Plugin authors should raise the minimum required CMake version to 3.15 to
+  ensure compatibility with new CMake scaffolding included in this
+  release. Older versions will trigger a warning at configuration time and,
+  depending on the functionality included in the plugin, may trigger subsequent
+  errors.
```

Comment:
```suggestion
  errors during configuration or the build.
```

Just making this clear that this is a build-time issue, not a run-time issue.

Reply:
Good one, yeah.